sdk/tables/azure-data-tables/samples/tables_entity_operations.cpp

```diff
@@ -12,26 +13,19 @@ using namespace Azure::Data::Tables;
 using namespace Azure::Data::Tables::Models;
```

Comment:
Using namespace Azure::Data::Tables::Models?
I think it should be just Azure::Data::Tables.

Reply:
removed , so it makes it clear where things come from